packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -406,7 +417,9 @@ export default class metadataBackup {
           })
 
           logger.error(
-            `Backuping pool metadata (${poolId}) for the remote (${remoteId}) has failed. (${job.id})`,
+            `Backuping pool metadata (${poolId}) for the remote (${remoteId}) has failed. (${
+              job.id
+            })`,
```

Comment:
How come those suddenly need line breaks?

Reply:
I think there is an issue with `prettier`.